WORKSPACE

```diff
@@ -25,7 +25,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Keep this version in sync with:
 #  * The BAZEL environment variable defined in .github/workflows/ci.yml, which is used for CI and nightly builds.
-versions.check(minimum_bazel_version = "4.2.2")
+versions.check(minimum_bazel_version = "4.2.2", maximum_bazel_version="4.5.0")
```

Comment:
Did you mean this to be 5.4.0? AFAIK bazel 5 was fine?

Reply:
oops, good catch! thx!